bin/create-gutenberg-test-post.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Is this script needed? It clones Gutenberg if it's not present, then runs [create-gutenberg-test-post.php](https://github.com/Automattic/amp-wp/pull/1018/files#diff-2520e4e1c81073b0b11d09153c68d994R1).

Reply:
It doesn't hurt, but it probably isn't necessary. The `amp_create_gutenberg_test_post()` function could just throw an exception if Gutenberg isn't active (e.g. if `! function_exists( 'do_blocks' )`)